components/src/dynamo/vllm/handlers.py

```
@@ -1964,6 +2046,38 @@ async def load_lora(self, request=None):
+                                await self._publish_lora_card(lora_name, old_info.id)
+                                self._published_loras.add(lora_name)
+                            except Exception as e:
+                                logger.exception(
+                                    f"Failed to publish LoRA {lora_name} "
+                                    f"ModelDeploymentCard: {e}"
+                                )
+                                yield {
+                                    "status": "error",
+                                    "message": (
+                                        f"LoRA '{lora_name}' is loaded but "
+                                        f"discovery publish failed: {str(e)}"
```

Ruff (0.15.20)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)